packit/api.py

```diff
@@ -1438,14 +1437,20 @@ def push_and_create_pr(
         git_branch: str,
         repo: Union[Upstream, DistGit],
         sync_acls: bool = False,
+        pr_title_suffix: Optional[str] = "",
     ) -> PullRequest:
         # the branch may already be up, let's push forcefully
         try:
             repo.push_to_fork(repo.local_project.ref, force=True, sync_acls=sync_acls)
         except PackitException as exc:
             logger.error(f"Push to fork failed: {exc}")
             raise
-        pr = repo.existing_pr(pr_title, git_branch, repo.local_project.ref)
+        pr = repo.existing_pr(
+            pr_title.rstrip(f" {pr_title_suffix}"),
```

Comment:
Why isn't branch enough here? :eyes:

Reply:
I don't follow you, I need to look for old created PRs but I don't know for which versions they are being created thus I remove the suffix  (== a version number) from the search.

Reply:
I wonder, do we actually need to include title in the search?

Reply:
+1, I would say that was Mato's point, that now that there is only one PR being reused, we do not have to check the title anymore

Reply:
Even though there could be both a propose downstream PR and a pull from upstream PR?

Reply:
Isn't title the same in such case and only source branch differs?

Reply:
Yes, true. The title is the same.
Ok then I will not match the title.